vllm/attention/backends/flash_attn.py

```diff
@@ -462,9 +462,19 @@ def build(self, seq_lens: List[int], query_lens: List[int],
             # The shape of graph_block_tables is
             # [max batch size, max context len // block size].
             input_block_tables = self.runner.graph_block_tables[:batch_size]
+            max_blocks = input_block_tables.shape[1]
             for i, block_table in enumerate(self.block_tables):
                 if block_table:
-                    input_block_tables[i, :len(block_table)] = block_table
+                    num_blocks = len(block_table)
```

Comment:
Do we need this for other attn backends?

Reply:
in general yes, but multistep works for flash_attn only currently, so we can incrementally add the support.